change-notes/1.20/analysis-javascript.md

```diff
@@ -0,0 +1,24 @@
+# Improvements to JavaScript analysis
+
+## General improvements
+
+* TODO
+
+## New queries
+
+| **Query**                                     | **Tags**                                             | **Purpose**                                                                                                                                                                 |
+|-----------------------------------------------|------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Useless comparison test | correctness | Highlight code that is unreachable due to a numeric comparison that is always true or alway false. |
```

Comment:
Are we happy with the use of the word "Useless" here?

Reply:
Yes, that's what the same query is called in the Java query suite. Please see my earlier comments on this.